packages/design/src/Indicator/Indicator.jsx

```diff
@@ -66,14 +66,14 @@ Indicator.defaultProps = {
 };
 
 const StyledSpinner = styled(SpinnerIcon)`
-  ${({ fontSize = '32px' }) => `
+  ${({ fontSize = '60px' }) => `
```

Comment:
Please verify that the fontSize changes does not affect other screens as well.
Use the IDEM-2017 ticket for the PR